README.md

```diff
@@ -1,15 +1,16 @@
 
 <!-- README.md is generated from README.Rmd. Please edit that file -->
 
-# rlistings <a href='https://github.com/insightsengineering/rlistings'><img src="man/figures/logo.png" align="right" height="200" width="200"/></a>
+# rlistings
 
 <!-- start badges -->
+
 [![Check
-🛠](https://github.com/insightsengineering/rlistings/actions/workflows/check.yaml/badge.svg)](https://insightsengineering.github.io/rlistings/main/unit-test-report/)
+🛠](https://github.com/insightsengineering/rlistings/actions/workflows/check.yaml/badge.svg)](https://github.com/insightsengineering/rlistings/actions/workflows/check.yaml)
```

Comment:
I just rebuilt it. I guess it was wrongly changed here instead that the quarto Rmd. Should I put them back?

Reply:
this link was right, right? https://insightsengineering.github.io/rlistings/main/unit-test-report/

@pawelru @cicdguy

Reply:
Thanks @cicdguy , i am a bit inclined to keep the unittest-test-report @Melkiades